app/components/sidebars/main/channels_list/filtered_list/filtered_list.js

```diff
@@ -59,10 +59,25 @@ class FilteredList extends Component {
         pastDirectMessages: [],
     };
 
-    constructor(props) {
+    static contextTypes = {
+        intl: intlShape.isRequired,
+    };
+
+    constructor(props, context) {
         super(props);
+
+        this.keyboardDismissProp = Platform.select({
+            android: {
+                onScrollBeginDrag: Keyboard.dismiss,
+            },
+            ios: {
+                keyboardDismissMode: 'on-drag',
+            },
+        });
+
+        this.intl = context.intl;
```

Comment:
Why save the `this.intl` when we could use `this.context.intl` in other places?

Reply:
don't know what is going on but this context seems to be null elsewhere other than the constructor, so I went with this.

Reply:
reverted this to use props